rpmlb/builder/base.py

```diff
@@ -43,13 +43,16 @@ def get_instance(cls, name):
         return instance
 
     def run(self, work, **kwargs):
-        self.before(work, **kwargs)
-
         is_resume = 'resume' in kwargs and kwargs['resume']
         resume_num = 0
         if is_resume:
             resume_num = kwargs['resume']
 
+        if is_resume:
+            LOG.info('Skip the process before build for resume option.')
```

Comment:
Your recognition is right. I think that your sentence is better than my sentence to understand easily and accurately.